### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by csadorf**
List aiidalab-launch before running the image manually.

### CODE REVIEW

File: docs/source/usage/index.rst [hunk 1]
```diff
@@ -21,8 +21,8 @@ AiiDAlab launch
 
 To run AiiDAlab on your own workstation or laptop you can either
 
- - run the image directly with: ```docker run aiidalab-docker-stack -p 8888:8888```, or
- - *(recommended)* use the :ref:`aiidalab-launch <usage:aiidalab-launch>` tool which is a thin docker wrapper.
+ - *(recommended)* Use the :ref:`aiidalab-launch <usage:aiidalab-launch>` tool which is a thin docker wrapper, or
+  - run the image directly with: ```docker run aiidalab-docker-stack -p 8888:8888```.
```
Comment (unkcpz) [inline docs/source/usage/index.rst:25]:
```suggestion
- run the image directly with: ```docker run aiidalab-docker-stack -p 8888:8888```.
```

**Review by unkcpz — CHANGES_REQUESTED** [review]
@csadorf thanks! Seems a minor indent bug fails the test.

**Comment by csadorf** [comment]
> @csadorf thanks! Seems a minor indent bug fails the test.

Hmm, not sure. I applied your suggestion and the indentation now seems off.

**Comment by unkcpz** [comment]
My bad, it should be an indent there in the second line. But not sure what caused the issue.

**Comment by csadorf** [comment]
@unkcpz Should be fixed now.

**Review by unkcpz — APPROVED** [review]
@csadorf thanks! 

What happened explicitly that fail the test?

**Comment by csadorf** [comment]
> @csadorf thanks!
> 
> What happened explicitly that fail the test?

Not sure to be honest. I edited the file directly on my computer instead of the GitHub IDE and then everything was fine.

**Pull request merged**